roles/uperf-bench/templates/configmap_script.yml.j2

```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: uperf-run-script-{{item.status.interfaces[0].ipAddress}}-{{ trunc_uuid }}
+  namespace: '{{ operator_namespace }}'
+data:
+  run_script.sh : |
+    export h={{item.status.interfaces[0].ipAddress}}
```

Comment:
Add `export kind=[container/vm]` so we can tell the difference between. We would need a SNAFU change too.

Reply:
i'm using the resourcetype argument over environment variable for this.